javascript-data-model/order-history.js

```diff
@@ -6,14 +6,16 @@ var orderHistory = [
     shipTo: 'JS Masher',
     orderNumber: 11439416898772232,
     items:
-    {
-      deliveryDate: 20200808,
-      deliveryMethod: null,
-      name: 'JavaScript for impatient programmers',
-      author: 'Ruschmayer, Dr. Axel',
-      returnCloseDate: 20200907,
-      price: 31.55
-    }
+    [
+      {
+        deliveryDate: 20200808,
+        deliveryMethod: null,
+        name: 'JavaScript for impatient programmers',
+        author: 'Ruschmayer, Dr. Axel',
+        returnCloseDate: 20200907,
```

Comment:
This should be on the main order object, too, for the same reason.

Reply:
Just to clarify, the name and author should stay with the item object, right? Also, for deliveryDate, deliveryMethod, and returnCloseDate, those will be unique to the item if the items in the order are delivered on different days, yes?